lib/sidekiq/dynamic_queues/attributes.rb

```diff
@@ -82,6 +89,12 @@ def expand_queues(queues)
         while q = queue_names.shift
           q = q.to_s
 
+          if q =~ /^(.+)\/([0-9]+)/
```

Comment:
Since you're not using the first capture group here, you could change it to `(?:.+)` and use `$1` down below.  You could also use `\A` instead of `^` to match the beginning he string instead of the beginning of the line (which is technically safer although likely not an issue here).